azext_iot/central/params.py

```diff
@@ -148,3 +148,11 @@ def load_central_arguments(self, _):
             help="Maximum number of messages to recieve from target device before terminating connection."
             "Use 0 for infinity.",
         )
+
+    with self.argument_context("iot central dps-keygen") as context:
```

Comment:
is this supposed to be at the top level? I agree with having it at the top level since its so different from our other commands and to emphasize it being offline, but the code is structured such that it looks like it belongs under the 'device' command-subgroup